docs/building-a-form/available-components.md

```diff
@@ -170,9 +169,35 @@ A group of options where the user can only select a single item.
 
 #### Usage guidelines
 
-In `formConfig`, define this in the data definition.
+The data for a group of radio buttons will be quite similar to the data for a select field (i.e., `string` type with an `enum` property), which means the `SelectWidget` will by default be rendered.
+
+To override the `SelectWidget`, pass `'ui:widget': 'radio'` to your `uiSchema` for that field. To specify different label text for each option, pass `'ui:options'` to `uiSchema`.
+
+Your config for a question where the answer is selected from a group of radio buttons might look like this:
+```
+schema: {
+  type: 'object',
+  properties: {
+    favoriteAnimal: {
+      type: 'string',
+      enum: ['dog', 'cat', 'octopus', 'sloth']
+    }
+  }
+},
+uiSchema: {
+  'ui:widget': 'radio',
+  'ui:options': {
+    labels: {
+      dog: 'Dog',
+      cat: 'Cat',
+      octopus: 'Octopus',
+      sloth: 'Sloth'
+    }
+  }
+}
+```
 
-For the code implementation, see [`RadioWidget`](https://github.com/usds/us-forms-system/blob/master/src/js/widgets/RadioWidget.jsx).
+For more information about this widget, see [`RadioWidget`](https://github.com/usds/us-forms-system/blob/master/src/js/widgets/RadioWidget.jsx).
```

Comment:
Were you for or against my reasoning for using the explicit "For the code implementation..." phrasing in https://github.com/usds/us-forms-system/pull/166#discussion_r203087358? I'm guessing this doesn't feel explicit to you, in which case, do you want to just remove the links to these files altogether? It's entirely possible it's just too much information in this context.